jsonnet/components/node-exporter.libsonnet

```
@@ -3,6 +3,26 @@ local textfileVolumeName = 'node-exporter-textfile';
 local tlsVolumeName = 'node-exporter-tls';
 local wtmpPath = '/var/log/wtmp';
 local wtmpVolumeName = 'node-exporter-wtmp';
+local configDir = '/var/node_exporter/config';
+local configVolumeName = 'node-exporter-config';
+local acceleratorsConfigFileName = 'accelerators-collector-config.yaml';
```

```suggestion
local acceleratorsConfigFileName = 'config.yaml';
```
see other change suggestions.

---

fixed